Flutter app fails to compile because of the older version of Kotlin in your library.

```
FAILURE: Build failed with an exception.

* What went wrong:
The Android Gradle plugin supports only Kotlin Gradle plugin version 1.5.20 and higher.
The following dependencies do not satisfy the required version:
project ':upi_payment_qrcode_generator' -> org.jetbrains.kotlin:kotlin-gradle-plugin:1.3.50
```

We are using v1.7.10 of the Kotlin which is latest currently along with v7.3.1 of the AGP in the Android module of our Flutter project.

Please upgrade your Kotlin version to the latest one possible. This is issue is blocking us to upgrade our Android project with the latest version of AGP, and other dependencies.

--- a/android/build.gradle
+++ b/android/build.gradle
@@ -2,7 +2,7 @@ group 'com.example.upi_payment_qrcode_generator'
 version '1.0-SNAPSHOT'
 
 buildscript {
-    ext.kotlin_version = '1.3.50'
+    ext.kotlin_version = '1.5.31'
     repositories {
         google()
         mavenCentral()

--- a/example/android/build.gradle
+++ b/example/android/build.gradle
@@ -1,5 +1,5 @@
 buildscript {
-    ext.kotlin_version = '1.3.50'
+    ext.kotlin_version = '1.5.31'
     repositories {
         google()
         mavenCentral()

